lol/lcuapi/Getting started with the LCU websocket.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Getting started with the LCU websocket"
+authors: "@thomasmarton"
+tags: "beginners-guide,lcu,websocket"
+markers: "generic"
+---
+# Getting started with the LCU websocket
+
+In LCU land, rather than the UX process constantly polling the LCU endpoints to see if anything has changed, the LCU API also hosts a websocket allowing the UI to be notified about changes (like you receiving a friend request or a chat message). This guide will show you how you can interact with this websocket to receive notifications that might be needed for your app (like champion select events or winning/losing).
+
+This guide assumes that you have basic knowledge about the LCU API. Check out our other guide to help you get started.
+
+---
+
+The LCU websocket uses the WAMP 1.0 protocol on this websocket. An example implementation in JavaScript can be found [over here](https://gist.github.com/Pupix/eb662b1b784bb704a1390643738a8c15).
```

Comment:
Should credit Pupix here as it is his gist.

Reply:
Pushed a change where he's properly credited